src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
```

Comment:
Instead of creating three separate instances of the same class, use one instance for your purposes